pkgs/by-name/sd/SDL2/package.nix

```
@@ -160,15 +159,19 @@ stdenv.mkDerivation (finalAttrs: {
       OpenGL
     ];
 
-  enableParallelBuilding = true;
-
   configureFlags =
     [ "--disable-oss" ]
     ++ lib.optional (!x11Support) "--without-x"
     ++ lib.optional alsaSupport "--with-alsa-prefix=${alsa-lib.out}/lib"
     ++ lib.optional stdenv.hostPlatform.isWindows "--disable-video-opengles"
     ++ lib.optional stdenv.isDarwin "--disable-sdltest";
 
+  dontDisableStatic = if withStatic then 1 else 0;
```

Isn't nix converting this truthy, so we can do

```suggestion
  dontDisableStatic = withStatic;
```

---

Outdated - done.